_pages/cfp.md

```diff
@@ -1,12 +1,145 @@
 ---
 layout: page
 title: CFP
-subtitle: Call for Proposals
+subtitle: StarCon 2018 Call for Proposals
 permalink: /cfp/
 ---
 
-Our call for proposals (CFP) for lightning talks will open on August 14 2017 and close on September 15 2017!
 
-All information about our CFP will be posted here.
+We’re excited to announce that StarCon is having it’s first Call for Proposals (CFP)! We’re looking for folks who have a passion for technology, learning and sharing to submit talks about their favourite piece(s) of technology.
+
+We’ve named this conference StarCon (*Con) for a reason. We want to represent many different people and topics in technology, just like how the regex asterisk symbol represents any character or symbol.
```

Comment:
you need to escape the `*` here, like `\*` -- that's why the source diff is italicized.